test/component/common/testHelpBlockContent.js

```diff
@@ -0,0 +1,26 @@
+const common = require('app/config');
+const {assert} = require('chai');
+
+class importedTest {
```

Comment:
As we will try and grow these tests, maybe we could call this class something more specificly related to the Help Block?

Reply:
No problem, so all further tests can be added to this class then?

Reply:
I agree with Tom, we currently have a class of `importedTest` in a file named `testHelpBlockContent.js`. The class name should reflect the file name and in this case should be called `TestHelpBlockContent`.

I don't think we need `importTest()` in `TestWrapper` as we can require `TestHelpBlockContent` directly from the test file and use it from there.